py2pack/__init__.py

```diff
@@ -221,7 +221,10 @@ def generate(args):
         data['source_url'] = _get_source_url(args.name, url['filename'])
     else:
         data['source_url'] = args.name + '-' + args.version + '.zip'
-    data['year'] = datetime.datetime.now().year                             # set current year
+
+    now = datetime.datetime.now()
```

Comment:
Why did you change this? AFAICS you never use it in the fedora template..

Reply:
I didn't feel like calling now() twice for no particular reason.

The 'year' was already initialized by the existing init ... that is not changed. (there is data['year'] = now.year) 

I am adding template of 'changelog_date' in a format suitable for Fedora changes.
The 'changelog_date' is used in the spec file template.